test/e2e/incremental-security.spec.js

```diff
@@ -134,22 +135,21 @@ describe('MetaMask', function () {
       await driver.delay(regularDelayMs);
       await driver.clickElement('#send');
 
-      const txStatus = await driver.findElement('#success');
-      await driver.wait(until.elementTextMatches(txStatus, /Success/u), 15000);
+      await driver.waitForSelector(
+        { css: '#success', text: 'Success' },
+        { timeout: 15000 },
+      );
     });
 
     it('switches back to MetaMask', async function () {
       await driver.switchToWindow(extension);
     });
 
     it('should have the correct amount of eth', async function () {
-      const balances = await driver.findElements(
-        '.currency-display-component__text',
-      );
-      await driver.wait(until.elementTextMatches(balances[0], /1/u), 15000);
-      const balance = await balances[0].getText();
-
-      assert.equal(balance, '1');
```

Comment:
I believe the intent of this assertion was to ensure this test doesn't pass if the balance is a number _containing_ the digit 1 (e.g. `1.51`, `100`, etc.). It might be worth preserving this assertion, both here and below where a similar assertion was removed.

Reply:
will do! Thanks